optaplanner-persistence/optaplanner-persistence-jpa/src/test/java/org/optaplanner/persistence/jpa/impl/score/AbstractScoreHibernateTypeTest.java

```diff
@@ -46,9 +48,9 @@
 
     @Before
     public void setUp() throws Exception {
-        context = PersistenceUtil.setupWithPoolingDataSource("optaplanner-persistence-jpa-test");
+        context = OptaplannerPersistenceUtil.setupWithPoolingDataSource(OptaplannerPersistenceUtil.OPTAPLANNER_PERSISTENCE_UNIT_NAME);
```

Comment:
Extracting this lookup name to a separate, new class is just verbosity as it's only used once.
We can always do taht later if it's ever used twice (which won't be the case).

Reply:
Agreed, will change it so there is no new class.